librabft-v2/src/data_sync.rs

```diff
@@ -46,7 +46,7 @@ pub struct DataSyncRequest {
     known_quorum_certificates: BTreeSet<Round>,
 }
 
-#[derive(Eq, PartialEq, Debug, Serialize, Deserialize)]
+#[derive(Eq, PartialEq, Debug, Serialize, Deserialize, Clone)]
```

Comment:
Do we need this `derive(Clone)` strictly speakign? Tokio channels don't seem to require it.